plugins/sentry/src/extensions.tsx

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import {
+  createComponentExtension,
+  createRoutableExtension,
+} from '@backstage/core';
+import { useEntity } from '@backstage/plugin-catalog';
+import { plugin, rootRouteRef } from './plugin';
+
+export const EntitySentryContent = plugin.provide(
+  createRoutableExtension({
+    mountPoint: rootRouteRef,
+    component: () =>
+      import('./components/SentryIssuesWidget').then(
```

Comment:
Hm so, this is a non-lazy, page-wrapped widget? 🤔

Reply:
Oh I see, it's just confusingly named Page, maybe rename that then

Reply:
It's a page, same content as the old `Router` component